test/tools/runner/config.js

```diff
@@ -251,6 +256,11 @@ class TestConfiguration {
       }
     }
 
+    if (this.isServerless) {
```

Comment:
what happens if the authSource is already set by the test? (e.g., in L236) Do we still want to override? Should this have a check whether ssl/authSource are already set and only update them in that case?

Reply:
None of the tests required for serverless deal with auth in any way, so this should be fine for this iteration of the testing - that said, it couldn't hurt to future-proof the code now rather than dealing it with if/when such tests are added to the serverless suite. We can't really validate it works though, unless such tests are added; do you think it's worth doing now?

Reply:
would it make sense to put this block as an `else if` to the one in L224? that way, if there are tests added that have the auth props set and they end up not being compatible with serverless, the test will fail and we'll be able to make an informed decision how to proceed at that point